library/ssl_cli.c

```diff
@@ -344,19 +350,6 @@ static int ssl_write_supported_elliptic_curves_ext( mbedtls_ssl_context *ssl,
     if( elliptic_curve_len == 0 )
         return( MBEDTLS_ERR_SSL_BAD_CONFIG );
 
-    MBEDTLS_SSL_CHK_BUF_PTR( p, end, 6 + elliptic_curve_len );
-
-    elliptic_curve_len = 0;
-
-    for( grp_id = ssl->conf->curve_list;
-         *grp_id != MBEDTLS_ECP_DP_NONE;
-         grp_id++ )
-    {
-        info = mbedtls_ecp_curve_info_from_grp_id( *grp_id );
-        elliptic_curve_list[elliptic_curve_len++] = MBEDTLS_BYTE_1( info->tls_id );
```

Comment:
@ronald-cron-arm Just so I understand it: I see the use of `MBEDTLS_BYTE_X` in this code, but not `MBEDTLS_PUT_UINT16_BE` as done by @brett-warren-arm in the new code. Was this a slip and Brett's version is what we want, or was there a reason not to use the `PUT_...` macro here before?